prisma-fmt/tests/text_document_completion/scenarios/extended_indexes_basic/schema.prisma

```diff
@@ -0,0 +1,67 @@
+generator js {
+  provider        = "prisma-client-js"
+  previewFeatures = ["fullTextIndex", "extendedIndexes"]
+}
+
+datasource db {
+  provider = "mysql"
+  url      = env("DATABASE_URL")
+}
+
+model Fulltext {
+  id      Int    @id
+  title   String @db.VarChar(255)
+  content String @db.Text
+  
+  @@fulltext()
+  @@fulltext([title, content], )
+}
+
+model Id {
+  
+  id String @id() @db.VarChar(3000)
+}
+
+model IdWithLength {
+  id String @id(length: 100) @db.VarChar(3000)
+}
+
+model Unique {
+  unique Int @unique()
+}
+
+model CompoundId {
+  id_1 String @db.VarChar(3000)
+  id_2 String @db.VarChar(3000)
+
+  @@id([id_1(length: 100), id_2(length: 10)])
+}
+
+model CompoundUnique {
+  unique_1 Int
+  unique_2 Int
+
+  @@unique([unique_1(sort: Desc), unique_2])
+}
+
+model Post {
+  title      String   @db.VarChar(300)
+  abstract   String   @db.VarChar(3000)
+  slug       String   @unique(sort: , length: 42) @db.VarChar(3000)
+  slug2      String   @unique() @db.VarChar(3000)
+  author     String
+  created_at DateTime
+
+  @@id([title(length: 100, ), abstract()])
+  @@index([author, created_at(sort: <|>)])
+  @@index([author, ])
+  @@index([])
+}
+
+model Example {
+  id    Int @id
+  value Int
+
+  @@index([value], type: Hash)
+  // @@index([value], type: BTree, name: "sss")
```

Comment:
What's the comment for? And name for index is kinda deprecated

Reply:
The whole schema is copy-pasted from language-tools test. I can remove that to make it less confusing.

Reply:
I'm doing a new ast PR next, I'll do that there.